index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class='navContainer'>
+        <div class="wrapper">
+            <nav>
+                <div class="logo">
+                    Ivan
+                </div>
+                <ul class="navItems">
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a class="navBtnMessage" href="#">
+                            <img class="messageBtn" src="images/mail.png" alt="">
+                        </a>
+                    </li>
+                </ul>
+                <ul class="navMenu">
+                    <li>
+                        <a class="navBtnMenu" href="#">
+                            <img class="menuBtn" src="images/menuUnion.png" alt="">
+                        </a>
+                    </li>
+                </ul>
+            </nav>   
+        </div>
+    </div>
```

Comment:
- [ ] This section should be inside the `header` tag. Kindly fix this issue.
- Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.